tools/ci/ethereum_test

```diff
@@ -27,8 +27,6 @@ cd $SRCROOT/integration
 deploy_v05_contracts
 ./service_agreement_test
 
-deploy_v06_contracts
-./flux_monitor_test
```

Comment:
Can we remove this code entirely now?

Reply:
It's removed in https://github.com/smartcontractkit/chainlink/pull/2524. Can you rebase?

Reply:
Definitely.  Ryan's TS integration test covers the old Bash stuff 100%.